ydb/library/yql/providers/generic/provider/yql_generic_load_meta.cpp

```diff
@@ -226,14 +167,70 @@ namespace NYql {
         }
 
     private:
-        const TStructExprType* ParseTableMeta(const NConnector::NApi::TSchema& schema, const std::string_view& cluster,
-                                              const std::string_view& table, TExprContext& ctx, TVector<TString>& columnOrder) try {
+        std::optional<TIssues> HandleDescribeTableResponse(
```

Comment:
Not sure if it would be better, but it is possible to return just non-optional non-empty `TIssues` as error indicator

Reply:
fixed